distro/Makefile

```diff
@@ -24,20 +26,23 @@ dist:
 	# copying docs and scripts
 	mkdir -p ${DIST}
 	mkdir -p ${HZ_BIN}
+	mkdir -p ${HZ_ARTIFACTS}
+	mkdir -p ${HZ_CFG}
 	mkdir -p ${HZ_LIB}
 	cp src/README.txt ${DIST}
-	cp src/hazelcast ${HZ_BIN}
-	for f in ${HZ_BIN}/* ; do sed -i '.bak' 's/$${hazelcast_version}/${HAZELCAST_VERSION}/g' $$f ; done
-	rm -f ${HZ_BIN}/*.bak
+	cp src/bin/hazelcast ${HZ_BIN}
 	chmod +x ${HZ_BIN}/*
 
 get-artifacts: dist
 	# get cli artifacts
 	cd .. && mvn clean package && cd -
 	cp ../target/hazelcast-command-line-*-jar-with-dependencies.jar ${HZ_LIB}/hazelcast-command-line.jar
+	cp src/artifacts/hazelcast-management-center-*.war ${HZ_ARTIFACTS}
```

Comment:
I thought about this, but this time the binary won't exist when running/testing from IDE. Also, we will need to add this download separately to all package manager implementations. But when we keep them in the repo and distribution package, we only download the CLI distribution. But I am open to suggestions, WDYT?

Reply:
If the artifact is stable, standard, and publically available, I think we should just download it when needed

Reply:
When you say _"When needed"_, do you mean when the user tries to use `hz mc` command? or when installing to the local using a package manager?

Reply:
When installing. Deferring the download until `hz mc` is risky - most users would probably assume that the tool is ready to go at this point

Reply:
Yes, indeed. If you mean when installing, then there would be no difference. It will be again downloaded somewhere (from the MC repo, or CLI repo). But if we keep it in the CLI repo, then the artifact will be also ready for tests & development.

Reply:
I think that keeping the artifact in the CLI repo is not good for the maintenance. Each time, a new version of MC is released, you need to change the binary. Plus, I don't see any real advantages for storing it in the CLI repo. For the purpose of (integration) tests, you can just download them.

To me it's the same what we do in our Docker images. We could copy Management Center war to `hazelcast/management-center-docker`, but we don't do it. The same for Hazelcast.

Reply:
Ok, if we don't want to keep the artifact in the repo, we can handle it two ways: 

1. We can download it whilst running the Makefile and include the MC binary in the distribution package. But we will need to keep this prepared package in this repo's releases page to provide it for package managers. Then the binary will still be kept in the repo but this time in the release package.
2. We can download it whilst running the package manager installations. But then we'll need two Makefiles: one for the complete distribution, one for the package managers.

@leszko & @pivovarit, any other ideas?

Reply:
I think the first option. Download it when executing `make`. Package should already include it. 

> But we will need to keep this prepared package in this repo's releases page to provide it for package managers.

Cannot we just do what Dockerfile does, so download MC, extract it, and copy the jar?

Reply:
> Cannot we just do what Dockerfile does, so download MC, extract it, and copy the jar?

This means the second option, we should do it in package manager scripts. Additionally, we will need another Makefile to provide a distribution package which can be installed manually.